src/DynamoCoreWpf/ViewModels/Watch3D/DynamoEffectsManager.cs

```diff
@@ -16,49 +16,56 @@ namespace Dynamo.Wpf.ViewModels.Watch3D
     public class DynamoEffectsManager : DefaultEffectsManager
     {
         internal static readonly string DynamoMeshShaderName = "DynamoMeshShader";
+        internal static readonly string DynamoPointLineShaderName = "DynamoPointLineShader";
 
-        public DynamoEffectsManager() : base() {
+        public DynamoEffectsManager()
```

Comment:
no longer calling the base class constructor? Why not?

Reply:
It is called by default. You need to include it specifically only if you need to pass parameters.